UPGRADE-5.0.md

```diff
@@ -301,6 +301,11 @@ Intl
  * Removed `Intl::getLocaleBundle()`, use `Locales` instead
  * Removed `Intl::getRegionBundle()`, use `Countries` instead
 
+Lock
+----
+
+ * Removed `Factory`, use `LockFactory` instead
```

Comment:
this should be submitted on 4.4, and UPGRADE-4.4.md should mention it too

Reply:
see https://github.com/symfony/symfony/pull/32508/files